src/main/java/use_cases/create_curriculum/CreateCurriculumInteractor.java

```diff
@@ -9,14 +14,37 @@
  */
 public class CreateCurriculumInteractor implements CreateCurriculumInputBoundary {
 
+    /* ********** *\
+    *  Attributes  *
+    \* ********** */
+    private final CurriculumFactory curriculumFactory;
+    private final CreateCurriculumOutputBoundary outputBoundary;
+
+    /* ************ *\
+    *  Constructors  *
+    \* ************ */
+    public CreateCurriculumInteractor(CurriculumFactory curriculumFactory,
+                                      CreateCurriculumOutputBoundary outputBoundary) {
+        this.curriculumFactory = curriculumFactory;
+        this.outputBoundary = outputBoundary;
+    }
+
+    /* ************* *\
+    *  Functionality  *
+    \* ************* */
     /**
      * Executes the use case for creating a new Curriculum
      *
      * @param name the new Curriculum's name
      */
     @Override
     public void createCurriculum(String name) {
+        Schedule activeUserSchedule = InMemoryUser.getActiveUser().getSchedule();
+        Curriculum newCurriculum = curriculumFactory.create(name);
+        activeUserSchedule.addCurriculum(newCurriculum);
 
+        CurriculumModel curriculumModel = new CurriculumModel(newCurriculum.getName(), newCurriculum.getID());
+        this.outputBoundary.displayCurriculum(curriculumModel);
     }
 
```

Comment:
Nice work here, the steps are clearly written and well-seen. Overriding the method from the input boundary, then getting the active User schedule, and then adding a newly created curriculum to the schedule in that method. Good, neat code